src/SlackMessages.ts

```diff
@@ -253,3 +246,97 @@ export function rugButtonFrom(action: ButtonSpec, command: IdentifiableInstructi
     }
     return button;
 }
+
+interface StandardAttachmentDefaults {
```

Comment:
There is RugMessages.ts where something similar is already being attempted - maybe just replace/enhance what is done there with new standard messages?